releasenotes/notes/add-rep-delay-c97d5aa8fc9696da.yaml

```
@@ -2,12 +2,14 @@
 features:
   - |
     Support for ``rep_delay`` flag added when assembling/executing a ``qobj``. ``rep_delay``
-    denotes the time between program executions. It must be chosen from a list of ``rep_delays``
-    from the backend, accessed as ``backend.configuration().rep_delays``.
-
-    ``rep_delay`` only works on backends which allow for dynamic repetition time. This setting is
-    found from ``backend.configuration().dynamic_reprate_enabled``. If false, ``rep_time`` will be
-    used rather than ``rep_delay``. ``rep_time`` only allows users to specify the duration of a
-    program, rather than the delay between programs.
+    denotes the time between program executions. It will only be used if
+    ``backend.configuration().dynamic_reprate_enabled``. In this case, its value must be within
+    range defined by the backend as ``backend.configuration().rep_delay_range``. If the user does
+    not supply a ``rep_delay`` valued, a backend default
```

I think there's a typo in this sentence. Maybe:
```suggestion
    not supply a ``rep_delay`, the backend default
```